sharktank_models/action.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: IREE Sharktank Models Tests
+
+inputs:
+  MATRIX_NAME:
+    description: "Identifying name of the matrix variation"
+    required: true
+  SKU:
+    description: "Type of SKU to test"
+    required: true
+  CHIP:
+    description: "Type of chip to test"
+    required: true
+  VENV_DIR:
+    description: "Path to the virtual environment"
+  TARGET:
+    description: "Type of target to test"
+    required: true
+  GPU:
+    description: "Type of GPU to test"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository 
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2 
+      with:
+        clean: false # doesn't clean up the venv created before this composite action
+    
+    - uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+      with:
+        python-version: "3.11"
+
+    - name: Setting up virtual environment and install Python dependencies
+      shell: bash
+      run: |
+        source ${{ inputs.VENV_DIR }}/bin/activate
+        pip install -e ${GITHUB_ACTION_PATH}/sharktank_models
+        pip install -r ${GITHUB_ACTION_PATH}/sharktank_models/requirements-iree.txt
+    
+    - name: Run sharktank specific model tests
+      shell: bash
+      if: contains(inputs.MATRIX_NAME, "model")
+      run: |
+        source ${{ inputs.VENV_DIR }}/bin/activate
+        HIP_TARGET=${{ inputs.GPU }} pytest iree-test-suites/sharktank_models/ \
```

Comment:
Looks like `HIP_TARGET` here could be set in the environment the same way as below, instead of as a prefix to the pytest command.

```yml
      env:
        ROCM_CHIP: ${{ inputs.CHIP }}
        SKU: ${{ inputs.SKU }}
```